<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nImageLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>281</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="adaptObr">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Адапт. пороговая обр</string>
    </property>
   </widget>
   <widget class="QLabel" name="oImageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>281</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="procGrey">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Поэлементный серый</string>
    </property>
   </widget>
   <widget class="QPushButton" name="procRGB">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Поэлементный RGB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить картинку</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ocuMet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>331</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Лок. пороговая обработка (Бернсен)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="procHSV">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Поэлементный HSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="median">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>331</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Лок. пороговая обработка (Лок. медиана)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>684</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
